net-libs/tox/tox-9999.ebuild

```diff
@@ -77,6 +77,10 @@ src_configure() {
 	cmake_src_configure
```

Comment:
The collection of log-* USE flags are quite complicated. Is this just for runtime logging, or build time?

Could we use an environment variable or something instead? Or allow users to use the EXTRA_ECONF equivalent for cmake (the name escapes me right now)?

Reply:
The documentation seems to suggest that this option enables or disables the logging at the code level by using macros. I don't recognize thee EXTRA_ECONF reference, but I would be open to hear more about it.

Reply:
See make.conf(5) for information about EXTRA_ECONF, it is an environment variable for people to pass in options to econf. But I guess we can keep this for now to just get a newer version in.

I'm just not sure about keeping so many log use flags in future.